qCC/ccGraphicalSegmentationTool.cpp

```diff
@@ -931,11 +965,11 @@ void ccGraphicalSegmentationTool::segment(bool keepPointsInside, ScalarType clas
 
 		// we project each point and we check if it falls inside the segmentation polyline
 #if defined(_OPENMP)
-        omp_set_num_threads(omp_get_max_threads());
+		omp_set_num_threads(omp_get_max_threads());
```

Comment:
Not sure what is this change? To be restored?

Reply:
sorry, this was a wrong indentation